app/assets/javascripts/draw-forfeit.js

```diff
@@ -0,0 +1,138 @@
+$(function() {
+  $(".request-draw").click(function() {
+    var updateURL = $(event.currentTarget).data("update-url");
+    var color = $(event.currentTarget).data("color");
+    if (color == "black") {
+      $.ajax({
+        type: 'PUT',
+        url: updateURL,
+        dataType: 'json',
+        data: { game: { black_draw: true } },
+        success: function(data) {
+          alert("A draw has been requested!");
+          $.ajax({
+            type: 'PUT',
+            url: Routes.game_draw_path(game_id)
```

Comment:
Blargh. That was some syntax I found in one tutorial, but it looks like in the actual documentation they do want exact numbers... I'll take your suggestion and put the id in a data-attr!